### ISSUE TITLE
VIC-IV documentation: question: is there a 16 colour video (not sprite) mode?

### ISSUE THREAD

**Issue by lgblgblgb** [MEGA65/mega65-user-guide#60]
is there any 16 colour video mode (not sprites ...)? I think that would be extremely useful, as higher res video modes requires large amount of RAM, and often you don't need 256 colours too much, but 4 colours (MCM) is too few ... I remember I asked this already (a long time ago) then you said, it's not so much planned currently.

**Comment by gardners** [comment]
Yes, there is such a mode.  It is accessed via Full-Colour Text Mode, and then setting the "4 bits per pixel" bit in the colour RAM byte-pair for each character. That is, you can mix 16 and 256 colour characters. The 16-colour ones are 16x8 instead of 8x8.
